### ISSUE TITLE
symbolic link to usb drives not working reliably

### ISSUE THREAD

**Issue by candide10** [nextcloud/nextcloudpi#109]
I have two usb drives connected (one for backup) and I just rebooted.  The symbolic links are now reversed.  The symbolic link "ncdrive" is pointing to my backup drive "ncbackup" and vice versa.  You should be able to see that here in this screenshot:
![image](https://user-images.githubusercontent.com/30739786/29245710-e9bffeca-7f97-11e7-866b-440a33b88b07.png)

I have the auto mount function turned on.

### What version of NextCloudPi are you using? ( eg: v0.17.2 )
v0.18.6

### What is the base image that you installed on the SD card? ( eg: NextCloudPi_07-21-17 )
It's from about a month ago - that is mid to late July.

**Comment by candide10** [comment]
The nextcloudpi image that I'm using is: 6/28/17

BTW - I rebooted a few times and still had the problem.  Then I manually deleted and then recreated the symbolic links and then rebooted and now things are working.  But doesn't the actual drive mapping supposedly switch around on successive reboots?

**Comment by nachoparker** [comment]
Hi,

Thanks for reporting that. I am reviewing the automount feature. Initially it was just meant to be used with one drive, so that inexperienced linux users did not have to deal with fstab.

Now it's time to give it some more attention, as there are things to polish.

**Comment by candide10** [comment]
OK.  Thanks.

**Comment by nachoparker** [comment]
Ok, I fixed that issue along with other things in `v0.20.3`

Still, I am not very happy with the automount feature. Maybe we should go for something like `udiskie`

At this point, still only recommended for basic usage and unexperienced users.

 - Doesn't work well with a drive with several partitions
 - It duplicates directories under `/media/`

BTW, one of your drives (`sdb` adove) does not have a partition table. This is not recommended, even though often it works, but it is considered bad practice and I think it won't be detected after the changes I just made.
